### ISSUE TITLE
The luxuries showed on the bar are overlapped with the information and menu button

### ISSUE THREAD

**Issue by realliyifei** [civfanatics/CQUI_Community-Edition#302]
It happens when you controlled too many luxuries 

<img width="1726" alt="image" src="https://user-images.githubusercontent.com/32441766/118876419-399e4180-b8bb-11eb-97d5-3a9e313af43a.png">

**Comment by the-m4a** [comment]
Yeah, I've noticed this too.  I wonder if the luxuries shown up top can be put into a scroll panel which would take care of this... though it would not show all luxuries at once except for very high resolutions

**Comment by dylansan** [comment]
From what I recall of Civ V, strategic resources always showed on the top bar individually, but luxuries were collapsed into a tooltip that would only show on hover (of the happiness icon I believe). Something like that would be nice here.

**Comment by Infixo** [comment]
There is a mod Top Panel Extension (https://steamcommunity.com/sharedfiles/filedetails/?id=2847136410) that basically does that. It shows luxuries in a nice tooltip. You can switch off luxuries in CQUI and everything fits nicely. The mod also shows amenities and housing.
